src/ixdat/plotters/sec_plotter.py

```diff
@@ -0,0 +1,267 @@
+        )
+
+        dOD_series = measurement.calc_dOD(V_ref=V_ref, t_ref=t_ref)
+        axes[0] = self.spectrum_series_plotter.heat_plot(
+            field=dOD_series,
+            tspan=tspan,
+            xspan=wlspan,
+            ax=axes[0],
+            cmap_name=cmap_name,
+            make_colorbar=make_colorbar,
+        )
+        if make_colorbar:
+            pass  # TODO: adjust EC plot to be same width as heat plot despite colorbar.
+
+        axes[1].set_xlim(axes[0].get_xlim())
+
+        return axes
+
+    def plot_waterfall(
+        self,
+        measurement=None,
+        ax=None,
+        V_ref=None,
+        t_ref=None,
+        cmap_name="jet",
+        make_colorbar=True,
+    ):
+        """Plot an SECMeasurement as spectra colored based on potential.
+
+        The top panel is a heat plot with wavelength on y-axis and color representing
+        spectrum. At most one of V_ref and t_ref should be given, and if neither are
+        given the measurement's default reference_spectrum is used to calculate the
+        optical density.
+
+        This uses SpectrumSeriesPlotter.plot_waterfall()
+
+        Args:
+            measurement (Measurement): The measurement to be plotted, if different from
+                self.measurement
+            tspan (timespan): The timespan of data to keep for the measurement.
+            wlspan (iterable): The wavelength span of spectral data to plot
+            ax (matplotlib Axis): The axes to plot on. A new one is made by default.
+            V_ref (float): potential to use as reference for calculating optical density
+            t_ref (float): time to use as a reference for calculating optical density
+            cmap_name (str): The name of the colormap to use. Defaults to "inferno",
+                which ranges from black through red and orange to yellow-white. "jet"
+                is also good.
+            make_colorbar (bool): Whether to make a colorbar.
+        """
+        measurement = measurement or self.measurement
+        dOD = measurement.calc_dOD(V_ref=V_ref, t_ref=t_ref)
+
+        return self.spectrum_series_plotter.plot_waterfall(
+            field=dOD,
+            cmap_name=cmap_name,
+            make_colorbar=make_colorbar,
+            ax=ax,
+            vs=measurement.V_str,
+        )
+
+    def plot_vs_potential(
+        self,
+        measurement=None,
+        tspan=None,
+        vspan=None,
+        V_str=None,
+        J_str=None,
+        axes=None,
+        wlspan=None,
+        V_ref=None,
+        cmap_name="inferno",
+        make_colorbar=False,
+        **kwargs,
+    ):
+        """Plot an SECMeasurement in two panels with potential as x-asis.
+
+        The top panel is a heat plot with wavelength on y-axis and color representing
+        spectrum. At most one of V_ref and t_ref should be given, and if neither are
+        given the measurement's default reference_spectrum is used to calculate the
+        optical density.
```

Comment:
optical density could change to dO.D., to be consistent through all the comments and figures